python/GafferImageUITest/ImageGadgetTest.py

```diff
@@ -95,7 +97,14 @@ def testDestroyWhileProcessing( self ) :
 
 		w.setVisible( True )
 
-		self.waitForIdle( 1000 )
+		# If this computer doesn't support floating point textures, the ImageGadget will warn about
+		# this the first time it tries to render.  Don't fail because of this
+		with IECore.CapturingMessageHandler() as mh :
+			self.waitForIdle( 1000 )
+
+		if len( mh.messages ):
+			for m in mh.messages:
+				print( "IGNORING MESSAGE: " + m.context + " : " + m.message )
```

Comment:
Don't clutter the test output - a successful test should be silent. Instead, let's assert that there are either no messages, or the single message we expect.

Reply:
Addressed in a26b343